lms/djangoapps/instructor/views/api.py

```diff
@@ -221,6 +223,25 @@ def wrapped(*args, **kwargs):  # pylint: disable=missing-docstring
     return decorator
 
 
+def require_sales_admin(func):
```

Comment:
I had tried to do this but hit some roadblocks last week. Let me check again and see what I can do.